README.adoc

```diff
@@ -45,8 +45,35 @@ mvn pom-editor:add-dep -Dgav='org.junit.jupiter:junit-jupiter:5.9.2' -Dscope=tes
 
 ==== Confirm the changes
 
+When you perform the goal add-dep with the command below:
+
+[source,sh]
+$ mvn pom-editor:add-dep -Dgav='junit:junit:4.13'
```

Comment:
@GianDeManincor, as we're using the commands by the plugin prefix (`pom-editor:add-dep`) instead of the FQN(full qualified name, it means, `groupId`:`artifactId` format, like `br.org.soujave:pom-editor-maven-plugin:add-dep`) we should mention that we're assuming that the user has configured the plugin group `br.org.soujava` into the `settings.xml`.

Reply:
@GianDeManincor maybe, we could add an information disclaimer like this:

> We assume you added `br.org.soujava`groupId into the `<pluginGroups>` at your `settings.xml` file. That's the reason why we're using the prefix plugin name `pom-editor`. Otherwise, you should replace it by `br.org.soujava:pom-editor-maven-plugin`.

Reply:
Ok, I'll change this part, but I'd not like to repeat this multiple times, because on **Add a dependency** we are explaining it. What is your suggestion?

Reply:
> @GianDeManincor maybe, we could add an information disclaimer like this:
> 
> > We assume you added `br.org.soujava`groupId into the `<pluginGroups>` at your `settings.xml` file. That's the reason why we're using the prefix plugin name `pom-editor`. Otherwise, you should replace it by `br.org.soujava:pom-editor-maven-plugin`.

It's a good idea. Maybe, we can put the settings.xml information at the end of the **How to use it**.

For example...

**How to use it**
|__ Add a dependency
|__ Confirm the changes
|__ Revert the changes
|__ More settings

Reply:
Hum... I'll suggest on the file, okay?